### PR TITLE
static fragment data passed to handlebars context

### PR DESCRIPTION
**Pull request by scypio**
Static fragment data passed to the handlebars. 

## Description

During FragmentContext initialization static data (prefixed with `data-const-` is now passed stored into the context. Prefix used for these properties is `_const`. Therefore data attribute `data-const-foo` can be referenced from handlebars as `{{_const.foo}]`

## Motivation and Context

Better code organization - and solution for some CMS systems that are not able to process the tags inside the fragment.

Implementation of https://github.com/Cognifide/knotx/issues/391

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code style](https://github.com/Cognifide/knotx/blob/master/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

---
I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

**Comment by scypio** [comment]
@Skejven changelog and upgrade notes updated :)

File: knotx-knot/knotx-knot-service/src/main/java/io/knotx/knot/service/impl/FragmentContext.java [hunk 1]
```diff
@@ -33,6 +37,7 @@
 
   private static final String DATA_SERVICE = ".*service.*";
   private static final String DATA_PARAMS = ".*params.*";
+  private static final String DATA_CONSTANTS = ".*const.*";
```
Comment (tomaszmichalak) [inline knotx-knot/knotx-knot-service/src/main/java/io/knotx/knot/service/impl/FragmentContext.java]: It should not be a part of Service Knot in my opinion.

File: knotx-knot/knotx-knot-service/src/main/java/io/knotx/knot/service/impl/FragmentContext.java [hunk 2]
```diff
@@ -100,6 +115,12 @@ private FragmentContext services(List<ServiceEntry> services) {
     this.services = services;
     return this;
   }
+  
+  private FragmentContext applyConstants(Map<String, String> constants) {
```
Comment (tomaszmichalak) [inline knotx-knot/knotx-knot-service/src/main/java/io/knotx/knot/service/impl/FragmentContext.java]: This change should be moved to Knot.x Data Bridge.

**Comment by tomaszmichalak** [comment]
This PR is outdated. Please check Knot.x 2.0 implementation to validate this idea.

**Pull request closed without merging**